.github/workflows/ci_tests.yml

```diff
@@ -31,12 +31,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        platform: [ubuntu-latest]
-        compiler:
-          - cpp: g++
+        platform:
+          - description: "ubuntu gcc"
+            cpp: g++
             c: gcc
```

Comment:
Do we ever use C compiler? More a curiosity.

Reply:
This is carried over from previous commits. 

We can evaluate this later

Reply:
> Do we ever use C compiler? More a curiosity.

Probably not us deliberately, but Cmake will look for one anyway, and any packages we fetch might use it. When this goes wrong it's surprisingly difficult to track down.

Reply:
Let's keep it.